atlas/AtlasMl/tests/test_pipelines.py

```
@@ -0,0 +1,220 @@
+        "Variables",
+        "Dictionaries",
+        "Functions",
+        "Loops",
+        "Tuples",
+        "Sets",
+        "Classes",
+        "Recursion"
+    ]
+    workflows.initial_texts(texts)
+    workflows.initial_competencies([
+        {"title": f"Data Structures Mastery",
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)